inscriptions/ISic000289.xml

```diff
@@ -185,18 +190,19 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#TNS">
+                <ab>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/><orig>ANSALO</orig><gap reason="lost" extent="unknown" unit="character"/><!-- Text 'dolphin tails' (cauda delphini) downwards from top right corner of preserved marble. -->
```

Comment:
fun. 
so, we deal with the text direction thus:
`<lb n="1" style="text-direction:circular-clockwise"/>`
cf. [https://epidoc.stoa.org/gl/latest/trans-linebreakdirection.html](https://epidoc.stoa.org/gl/latest/trans-linebreakdirection.html).
The note about dolphin tails we would add elsewhere in the file after the `<handDesc>` elements, thus:
` <decoDesc><decoNote>Dolphin tails on the lower left of the fragment</decoNote></decoDesc>`